data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -547,19 +547,19 @@
 
         [message]
             speaker=unit
-            message= _ "Who goes there! Come out of the house now!"
+            message= _ "Who goes there! Come out now!"
         [/message]
         [message]
             speaker=Aleron
-            message= _ "Stay! It’s me, Aleron! The villagers hid me from the bandits when the town was overrun. They have kept me safe, though I was gravely wounded..."
+            message= _ "Stay your hand! It’s me, Aleron! The villagers hid me from the bandits when the town was overrun. They kept me safe while I was gravely wounded."
         [/message]
         [message]
             speaker=Sir Gerrick
-            message= _ "I am glad to find you alive Aleron, we feared the worst when you did not return after the battle!"
+            message= _ "Thank the heavens! I am glad to find you alive, Aleron. We feared the worst when you did not return after the battle."
```

Comment:
"Thank the heavens!" assumes a very specific theological framework which does not seem to be supported by other Wesnothian lore. Omitting entirely might be best, but "Thank the light!" I think is more likely.